.env.test

```diff
@@ -5,3 +5,7 @@ DB_PORT=5432
 DB_USER=postgres
 DB_HOST=localhost
 DB_NAME=test
+DISCORD_CLIENT_ID=123
+NEXT_PUBLIC_DISCORD_CLIENT_ID=123
+DISCORD_CLIENT_SECRET=123
+DB_PASSWORD=password
```

Comment:
I'd try to keep these organized so they're easier to maintain. For instance, we should probably move `DB_PASSWORD` up to be with the other `DB_` vars. If we wanted to take it a step further, we could separate them into groups too. eg.

```ini
AUTH_SECRET=secret

DB_HOST=localhost
DB_PORT=5432
DB_NAME=test
DB_USER=postgres
DB_PASSWORD=password

DISCORD_CLIENT_ID=123
DISCORD_CLIENT_SECRET=123
DISCORD_REDIRECT_URI=http://localhost:3000/auth-redirect
DISCORD_GUILD_ID=735923219315425401

NEXT_PUBLIC_DISCORD_CLIENT_ID=123
```

Reply:
Agree with this, but I would instead recommend grouping it by what's a secret / what's not a secret.

Reply:
@timmyichen, do you meaning something like this?
```ini
# Secrets
AUTH_SECRET=secret
DISCORD_CLIENT_SECRET=123
DB_PASSWORD=password

# Configuration Values
DB_HOST=localhost
DB_PORT=5432
DB_NAME=test
DB_USER=postgres

DISCORD_CLIENT_ID=123
DISCORD_REDIRECT_URI=http://localhost:3000/auth-redirect
DISCORD_GUILD_ID=735923219315425401

NEXT_PUBLIC_DISCORD_CLIENT_ID=123
```

Reply:
yep!

Reply:
Sounds good, will make the change!